src/podman.scss

```diff
@@ -69,6 +69,26 @@
   color: white;
 }
 
+.ct-badge-toolbox {
+    background-color: var(--pf-v5-global--palette--purple-100);
+    color: var(--pf-v5-global--palette--purple-600);
+}
+
+.ct-badge-distrobox {
+    background-color: var(--pf-v5-global--palette--gold-100);
+    color: var(--pf-v5-global--palette--gold-600);
+}
+
+.pf-v5-theme-dark .ct-badge-toolbox {
+    background-color: var(--pf-v5-global--palette--purple-500);
+    color: white;
+}
+
+.pf-v5-theme-dark .ct-badge-distrobox {
+    background-color: var(--pf-v5-global--palette--gold-500);
+    color: white;
+}
```

Comment:
As this is SCSS instead of CSS, it's also possible to write it with nesting, so the associated CSS values are grouped by type, like this:

```scss
.ct-badge-toolbox {
    background-color: var(--pf-v5-global--palette--purple-100);
    color: var(--pf-v5-global--palette--purple-600);

    .pf-v5-theme-dark & {
        background-color: var(--pf-v5-global--palette--purple-500);
        color: white;
    }
}

.ct-badge-distrobox {
    background-color: var(--pf-v5-global--palette--gold-100);
    color: var(--pf-v5-global--palette--gold-600);

    .pf-v5-theme-dark & {
        background-color: var(--pf-v5-global--palette--gold-500);
        color: white;
    }
}
```

This is also the way you'd write plain CSS with nesting, which is a new feature inspired by SCSS. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting — but while native nesting is now available in all browsers (except Firefox extended support release), we can't use it yet. But we're using SCSS, so it's fine to use where we know it'll get compiled down. https://caniuse.com/css-nesting

It's not a big deal though; what's here in the PR is totally correct. :grin: